### ISSUE TITLE
No callback from the threadified function after minification

### ISSUE THREAD

**Issue by tondapup** [flozz/threadify#6]
Hi @flozz, Everything works perfectly fine in development. After building and minification for production. The threadify function is called but never hits success/errror callback on return. Am I missing something?

**Comment by flozz** [comment]
Hello,

I have to made some tests... it is probably a `global` var that is minified... I will have to write some tests against minification... :)

**Comment by tondapup** [comment]
@flozz : What will be the temporary fix for that? I am using this in react so I will have to build and minify for production.

**Comment by flozz** [comment]
Maybe you can try to replace `global` by `window` in `helpers.js`. If it does not work, I will have to do further investigations...

**Comment by tondapup** [comment]
@flozz : This does not work.

**Comment by flozz** [comment]
Hello,

I made some test, and my last commit should fix you issue: https://github.com/flozz/threadify/commit/7d2a2bc4564a4020812b6a0d96373f5254562cfd

can you test it and let me know if it is ok for you?

**Comment by flozz** [comment]
Hum yes, as I have not yet published the package, you have to install it from this repository instead of NPM.

In your package.json file, replace the threadify intry by this:

```json
{
    ...
    "threadify": "git+https://github.com/flozz/threadify.git",
    ...
}
```

**Comment by tondapup** [comment]
@flozz : You are awesome!! Thank you.

**Comment by flozz** [comment]
Released on NPM: **threadify 0.2.4**
